assignments/prototype-refactor.js

```diff
@@ -7,3 +7,106 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+/*
+  === GameObject ===
+  * createdAt
+  * name
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+class GameObject{
+    constructor (ObjStats){
+        this.createdAt = ObjStats.createdAt;
+        this.name = ObjStats.name;
+        this.dimensions = ObjStats.dimensions;
```

Comment:
The convention for JS object variables is to spell them in camelCase (first letter is in lowercase; first letter of each subsequent word is in uppercase). Doing this visually distinguishes it from other Javascript entities like classes and constructor functions.